app/src/components/Common/Link/index.tsx

```diff
@@ -0,0 +1,20 @@
+import { Link as MuiLink, LinkProps } from "@mui/material";
+
+type Ilink = {
```

Comment:
why you choose this name of the interface?
type - ExampleType
interface - IExample

Reply:
oh i got it? we need to name it LinkType
right?

Reply:
yes!

Reply:
may be LinkComponentType would be better?